src/libsyntax/parse/mod.rs

```diff
@@ -33,7 +33,6 @@ pub mod attr;
 
 pub mod common;
 pub mod classify;
-pub mod obsolete;
```

Comment:
I don't think it'd be useful at all after 1.0, but we may make some more syntax tweaks in the period before 1.0 where this would still be useful.